src/vdaf/prio3.rs

```diff
@@ -375,19 +375,37 @@ where
         prg.into_seed()
     }
 
+    fn random_size(&self) -> usize {
+        if self.typ.joint_rand_len() == 0 {
+            // Two seeds per helper for measurement and proof shares, plus one seed for proving
+            // randomness.
+            (usize::from(self.num_aggregators) * 2 - 1) * SEED_SIZE
```

Comment:
Lines up better with the comment, IMO:
```suggestion
            ((usize::from(self.num_aggregators) - 1) * 2 + 1) * SEED_SIZE
```